src/DOMElementFilter.ts

```diff
@@ -0,0 +1,259 @@
+/**
+ * Source: https://github.com/facebook/jest/blob/e7bb6a1e26ffab90611b2593912df15b69315611/packages/pretty-format/src/plugins/DOMElement.ts
+ */
+import type {Config, NewPlugin, Printer, Refs} from 'pretty-format'
```

Comment:
Requires https://github.com/facebook/jest/pull/11342

Might need to use module augmentation depending on the targetted release of https://github.com/facebook/jest/pull/11342